docs/dev_guide/contents/extending_fido.rst

```diff
@@ -289,6 +289,23 @@ An Example of ``register_values()``
 
         return adict
 
+Registering custom attrs in the ``attrs`` namespace
+---------------------------------------------------
+
+When you have custom attrs defined in a separate attrs module, you can add them to the namespace using the :meth:`~sunpy.net.BaseClient._attrs_module` class method.
+The method returns a tuple of length 2, where the first element is the name you want to add to the main attrs namespace. The second is module where the custom attrs exist, so that can be imported.
+An example for this can be seen as implemented in the JSOC client:
+
+.. code-block:: python
+
+    @classmethod
+    def _attrs_module(cls):
+        return 'jsoc', 'sunpy.net.jsoc.attrs'
+
+This adds the attrs that exist within ``sunpy.net.jsoc.attrs`` such as ``Keyword`` which can now be accessed by the man attrs import, e. g., ``a.jsoc.Keyword``.
+
+This import can be any module, external as well.
```

Comment:
I am not clear on what this sentence means? Does it need clarifying (or removal)?

Reply:
I was trying to see that this works for internal sunpy code, like the example but also for external libraries. Like DKIST.

Maybe we should remove it.

Reply:
does `The source module here need not be an internal SunPy module.` after the description of the second element of the tuple work?

Reply:
Yeah I'm fine with the new line, remove it here and I'm happy.